src/main/java/com/nike/cerberus/auth/connector/onelogin/OneLoginAuthConnector.java

```diff
@@ -157,7 +157,7 @@ protected SessionLoginTokenData verifyFactor(final String deviceId,
             String msg = String.format("stateToken: %s failed to verify 2nd factor for reason: %s",
                     stateToken, verifyFactorResponse.getStatus().getMessage());
 
-            if (verifyFactorResponse.getStatus().getCode() == 400L) {
+            if (verifyFactorResponse.getStatus().getCode() == 401) {
```

Comment:
Unfortunately, OneLogin changed what they're returning here without changing the API version.